src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderHealthProblem.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.JoinColumnOrFormula;
+import org.hibernate.annotations.JoinColumnsOrFormulas;
+import org.hibernate.annotations.JoinFormula;
+import org.hibernate.annotations.NotFound;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.time.LocalDate;
+
+import static org.hibernate.annotations.NotFoundAction.IGNORE;
+
+
+@Data
+@EqualsAndHashCode(callSuper=false, exclude = "charges")
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "OFFENDER_HEALTH_PROBLEMS")
+@ToString(of = {"id"})
+public class OffenderHealthProblem {
+
+    @Id
+    @Column(name = "OFFENDER_HEALTH_PROBLEM_ID", nullable = false)
+    private Long id;
```

Comment:
How is the id generated, I was expecting it to be a sequence

Reply:
used the example code above